### PR TITLE
VIH-9895 axe results exceptions fix

### PR DESCRIPTION
**Pull request by abbasiTahir**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-9895

### Change description ###

Fixed the issues where duplicate labels were causing the accessibility tests failures .

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/shared/select/select.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div>
-  <label class="govuk-label" for="select-menu">{{ title }}</label>
+  <label class="govuk-label" for="select-menu{{title}}">{{ title }}</label>
```
Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/shared/select/select.component.html:2]: I think you're missing a hyphen here - it should match the ng-select's id below

`select-menu-{{title}}`

Comment (abbasiTahir) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/shared/select/select.component.html]: thanks fixed.

**Review by phildebrah — APPROVED** [review]
Tested and Approved

**Pull request merged**